meson.build

```diff
@@ -0,0 +1,100 @@
+project(
+  'rauc-hawkbit-updater',
+  'c',
+  version : '1.2',
+  meson_version : '>=0.50',
+  default_options: [
+    'warning_level=3',
+  ],
+  license : 'LGPL-2.1-only',
+)
+
+conf = configuration_data()
+conf.set_quoted('PROJECT_VERSION', meson.project_version())
+
+libcurldep = dependency('libcurl', version : '>=7.47.0')
+giodep = dependency('gio-2.0', version : '>=2.26.0')
+giounixdep = dependency('gio-unix-2.0', version : '>=2.26.0')
+jsonglibdep = dependency('json-glib-1.0')
+
+incdir = include_directories('include')
+
+sources_updater = [
+  'src/rauc-hawkbit-updater.c',
+  'src/rauc-installer.c',
+  'src/config-file.c',
+  'src/hawkbit-client.c',
+  'src/json-helper.c',
+  'src/log.c',
+]
+
+c_args = '''
+  -Wformat-nonliteral
```

Comment:
This seems to be included in `-Wformat=2` below, see [GCC's warning options](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html).

Reply:
indeed :+1: